tests/test_handlers.py

```diff
@@ -350,84 +378,87 @@ def timeout_fn() -> None:
+        # occur, but also ensure time.sleep recieves a non-negative number.
+        time_to_last_timeout: float = max(0, (start_ts + expected_timeout_deltas[-1]) - time.time())
         time.sleep(time_to_last_timeout)
 
-        self.logger.log(logging.INFO, f"ensure close flushes correctly")
-
-        self.compare_all_logs()
+        self.logger.log(logging.INFO, "ensure close flushes correctly")
+        self.close_and_compare_logs()
         self.assertEqual(timeout_count.value, expected_timeout_count)
         for i in range(expected_timeout_count):
             self.assertAlmostEqual(
-                datetime.fromtimestamp(timeout_ts[i]),  # type: ignore
-                datetime.fromtimestamp(start_ts + expected_timeout_deltas[i]),
-                delta=timedelta(milliseconds=TIMESTAMP_DELTA_MS),
+                timeout_ts[i],  # type: ignore
+                start_ts + expected_timeout_deltas[i],
+                delta=ASSERT_TIMESTAMP_DELTA_S,
             )
 
     def test_pushback_soft_timeout(self) -> None:
-        delta_ms: int = 400
-        delta_s: float = delta_ms / 1000
+        delay: float = LOG_DELAY_S
+        delta_s: float = delay * 2
+        delta_ms: int = int(delta_s * 1000)
         self._test_timeout(
             loglevels=[logging.INFO, logging.INFO, logging.INFO],
-            delay=0.2,
+            delay=delay,
             hard_deltas={logging.INFO: 30 * 60 * 1000},
             soft_deltas={logging.INFO: delta_ms},
             # delay < soft delta, so timeout push back should occur
             # timeout = final log occurrence + soft delta
             expected_timeout_count=2,
             expected_timeout_deltas=[
-                (0.2 * 2) + delta_s,  # 2 logs * delay + soft delta
-                (0.2 * 2) + delta_s + 0.256,  # last timeout + 256ms pad
+                (2 * delay) + delta_s,  # 2 logs * delay + soft delta
+                (2 * delay) + delta_s + TIMEOUT_PADDING_S,  # last timeout + pad
             ],
         )
 
     def test_multiple_soft_timeout(self) -> None:
-        delta_ms: int = 100
-        delta_s: float = delta_ms / 1000
+        delay: float = LOG_DELAY_S * 2
+        delta_s: float = LOG_DELAY_S
+        delta_ms: int = int(delta_s * 1000)
         self._test_timeout(
             loglevels=[logging.INFO, logging.INFO, logging.INFO],
-            delay=0.2,
+            delay=delay,
             hard_deltas={logging.INFO: 30 * 60 * 1000},
             soft_deltas={logging.INFO: delta_ms},
             # delay > soft delta, so every log will timeout
             expected_timeout_count=4,
             expected_timeout_deltas=[
                 delta_s,  # soft delta
-                0.2 + delta_s,  # delay + soft delta
-                (0.2 * 2) + delta_s,  # 2 * delay + soft delta
-                (0.2 * 2) + delta_s + 0.256,  # last timeout + 256ms pad
+                delay + delta_s,  # delay + soft delta
```

Comment:
Instead of using comments to describe what the variables are, why not rename the variables to reflect the comments?